modules/virt-about-container-disks.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-containerdisk-volumes.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-container-disk-with-datavolumes.adoc
+
+[id="virt-about-container-disks_{context}"]
+= About container disks 
+
+A container disk is a virtual machine image stored as a container image in a container image registry. Container disks can deliver the same disk images to multiple virtual machines and can be used to create large numbers of virtual machine clones.
+
+A container disk can either be imported into a persistent volume claim (PVC) using a DataVolume and attached to a virtual machine, or attached directly to a virtual machine as an ephemeral `containerDisk` volume.
+
+== Importing a container disk into a PVC using a DataVolume
+
+The Containerized Data Importer (CDI) imports the container disk into a PVC using a DataVolume. The DataVolume can then be attached to a virtual machine for persistent storage.
+
+== Attaching a container disk to a virtual machine as a containerDisk volume
+
+A containerDisk volume is ephemeral. It is discarded when
+the virtual machine is stopped, restarted, or deleted. When a virtual machine with a containerDisk volume starts, the container image is pulled from the registry and hosted on the node that is hosting the virtual machine.
+
+A containerDisk volume is useful for read-only filesystems such as CD-ROMs or for disposable virtual machines.
```

Comment:
Even if most of this module stays in the passive voice, I think that this sentence in particular should be rephrased to something like "You can use a `containerDisk` volume..."